diff --git a/tensorflow/lite/ios/BUILD.apple b/tensorflow/lite/ios/BUILD
similarity index 98%
rename from tensorflow/lite/ios/BUILD.apple
rename to tensorflow/lite/ios/BUILD
index 2f6b40c5a8a..87a3d5cd2e4 100644
--- a/tensorflow/lite/ios/BUILD.apple
+++ b/tensorflow/lite/ios/BUILD
@@ -29,8 +29,7 @@ sh_binary(
         "hide_symbols_with_allowlist.sh",
     ],
     visibility = [
-        "//tensorflow/lite:__subpackages__",
-        "//tensorflow_lite_support:__subpackages__",
+        "//visibility:public",
     ],
 )
 
diff --git a/tensorflow/lite/ios/ios.bzl b/tensorflow/lite/ios/ios.bzl
index acb9cabff13..dcbc1a19a49 100644
--- a/tensorflow/lite/ios/ios.bzl
+++ b/tensorflow/lite/ios/ios.bzl
@@ -59,7 +59,7 @@ def tflite_ios_static_framework(
     cmd = ("INPUT_FRAMEWORK=\"$(location " + framework_target + ")\" " +
            "BUNDLE_NAME=\"" + bundle_name + "\" " +
            "ALLOWLIST_FILE_PATH=\"$(location " + allowlist_symbols_file + ")\" " +
-           "EXTRACT_SCRIPT_PATH=\"$(location //tensorflow/lite/ios:extract_object_files_main)\" " +
+           "EXTRACT_SCRIPT_PATH=\"$(location @org_tensorflow//tensorflow/lite/ios:extract_object_files_main)\" " +
            "OUTPUT=\"$(OUTS)\" " +
            "\"$(location //tensorflow/lite/ios:hide_symbols_with_allowlist)\"")
 
@@ -69,8 +69,8 @@ def tflite_ios_static_framework(
         outs = [name + ".zip"],
         cmd = cmd,
         tools = [
-            "//tensorflow/lite/ios:extract_object_files_main",
-            "//tensorflow/lite/ios:hide_symbols_with_allowlist",
+            "@org_tensorflow//tensorflow/lite/ios:extract_object_files_main",
+            "@org_tensorflow//tensorflow/lite/ios:hide_symbols_with_allowlist",
         ],
     )
 
